module2/intermed-css1.html

```diff
@@ -18,7 +18,7 @@
 		<div class="reveal">
 			<div class="slides">
 				<section>Welcome to the Free freeCodeCamp Bootcamp</section>
-				<section>Intermediate HTML & CSS</section>
+				<section>Intermediate HTML & CSS Pt 1</section>
```

Comment:
Let's match this section with the original listed in the main index.html Intermediate CSS Pt 2.